packages/babel-core/test/option-manager.js

```diff
@@ -27,14 +27,19 @@ describe("option-manager", () => {
       return { plugin, calls };
     }
 
-    it("should throw if a plugin is repeated", () => {
-      const { calls, plugin } = makePlugin();
+    it("should throw if a plugin is repeated, with information about the repeated plugin", () => {
+      const { calls, plugin } = makePlugin("my-plugin");
 
       expect(() => {
         loadOptions({
-          plugins: [plugin, plugin],
+          plugins: [
+            [plugin, undefined, "my-plugin"],
+            [plugin, undefined, "my-plugin"],
```

Comment:
What does it throw if you don't specify the `my-plugin` name here?

Reply:
@nicolo-ribaudo the example output in the pr is a case where no name is specified.  Because I'm serializing it with `JSON.stringify` you don't see the `name` key in the output at all.

The output for this test is:
```
      Duplicate plugin/preset detected.
      If you'd like to use two separate instances of a plugin,
      they need separate names, e.g.

        plugins: [
          ['some-plugin', {}],
          ['some-plugin', {}, 'some unique name'],
        ]

      Duplicates detected are:
      [
        {
          "name": "my-plugin",
          "alias": "base$0",
          "dirname": "/Users/dhamilto/src/babel/babel/packages/babel-core/test",
          "ownPass": false
        },
        {
          "name": "my-plugin",
          "alias": "base$1",
          "dirname": "/Users/dhamilto/src/babel/babel/packages/babel-core/test",
          "ownPass": false
        }
      ]
```

With the name omitted, the output becomes:

```
      Duplicate plugin/preset detected.
      If you'd like to use two separate instances of a plugin,
      they need separate names, e.g.

        plugins: [
          ['some-plugin', {}],
          ['some-plugin', {}, 'some unique name'],
        ]

      Duplicates detected are:
      [
        {
          "alias": "base$0",
          "dirname": "/Users/dhamilto/src/babel/babel/packages/babel-core/test",
          "ownPass": false
        },
        {
          "alias": "base$1",
          "dirname": "/Users/dhamilto/src/babel/babel/packages/babel-core/test",
          "ownPass": false
        }
      ]
```